src/math_services.py

```diff
@@ -1,19 +1,22 @@
-def double_list(integer_list):
+def multiply_list(integer_list, integer_value):
     """
-    Multiplies every integer in the list by 2.
+    Multiply every integer in the list by an integer value.
 
     Receives
     --------
     integer_list : list
-        List of integer values.
+        List of integers.
+
+    integer_value : integer
+        Integer value by which every element should be multiplied.
 
     Returns
     -------
-    doubled_list : list
-        List of integer values multiplied by 2.
+    multiplied_list : list
+        List of integers multiplied by input value.
     """
 
-    doubled_list = [2*x for x in integer_list]
+    multiplied_list = [integer_value*x for x in integer_list]
     integer_list.clear()
-    integer_list.extend(doubled_list)
-    return integer_list
+    integer_list.extend(multiplied_list)
+    return integer_list
```

Comment:
A falta de um linha em branco no final deste arquivo está gerando o erro `src/math_services.py:22:24: W292 no newline at end of file`.